sandboxes/pypi/analyze.py

```diff
@@ -53,13 +53,36 @@ def analyze(package):
 
 
 def main():
-    if len(sys.argv) != 2:
-        raise ValueError(f'Usage: {sys.argv[0]} package_name[==version]')
-
-    package_with_version = sys.argv[1]
-    pip_install(package_with_version)
+    args = list(sys.argv)
+    script = args.pop(0)
+
+    if len(args) < 2 or len(args) > 4:
```

Comment:
nit: The builtin [argparse](https://docs.python.org/3/library/argparse.html) makes all of this a lot easier. Feel free to add a TODO here instead though.

Reply:
FWIW I was specifically avoiding importing arg parsing libs to limit the number of side effects that appear in the strace.

I will add a comment indicating that.

Reply:
Ah, makes sense, thanks!